packages/@cdk-cosmos/core/src/ecs-solar-system/ecs-solar-system-core-stack.ts

```diff
@@ -75,17 +67,22 @@ export const EcsSolarSystemCoreStackBuilder = (
         ...clusterProps,
         clusterName: this.singletonId('Cluster'),
         vpc: this.vpc,
+        capacity:
+          clusterProps.capacity !== null
```

Comment:
Shouldn't this be  `clusterProps.capacity == null`

Reply:
nah the idea is if they done want the default asg then pass in null (explicit)

Reply:
undefined = default and something is merged in with defaults ...

Reply:
ahh...ok so by default ASG will be created

Reply:
i think if they want their own ASG then pass in null and create your own, atm we have no real requirement to have access to the ASG ... but when that changes, then in the future well change the props ?